app/components/post-search-results.js

```diff
@@ -0,0 +1,150 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { not } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+import { getQueryString } from '@wikia/ember-fandom/utils/url';
+
+import config from '../config/environment';
+
+// TODO: Remove this when all discussions' posts are in the index
+const QUIZZES_WHITELIST = [
+  // test communities
+  'keikosandbox.fandom.com',
+  'keiko-test.fandom.com',
+  'gameofthrones.bartosz.fandom-dev.us',
+  'gameofthrones.chriss.fandom-dev.us',
+  'xkxd02.mattk.fandom-dev.us',
+  'xkxd02.jrogan.fandom-dev.us',
+  'xkxd02.chriss.fandom-dev.us',
+  'xkxd02.bartosz.fandom-dev.us',
+  'keikosandbox.sandbox-s3.fandom.com',
+  'keikosandbox.sandbox-s4.fandom.com',
+  'keiko-test.sandbox-s3.fandom.com',
+  'keiko-test.sandbox-s4.fandom.com',
+  // FIXME: Enable when we want to launch on Quizzes' Communities
+  // 'gameofthrones.fandom.com',
+  // 'attackontitan.fandom.com',
+  // 'marvelcinematicuniverse.fandom.com',
+  // 'marvel.fandom.com',
+  // 'southpark.fandom.com',
+  // 'starwars.fandom.com',
+  // 'strangerthings.fandom.com',
+  // 'xmenmovies.fandom.com',
+  // 'arrow.fandom.com',
+  // 'bojackhorseman.fandom.com',
+  // 'dc.fandom.com',
+  // 'dcextendeduniverse.fandom.com',
+  // 'disney.fandom.com',
+  // 'fastandfurious.fandom.com',
+  // 'godzilla.fandom.com',
+  // 'lionguard.fandom.com',
+  // 'miraculousladybug.fandom.com',
+  // 'riverdale.fandom.com',
+  // 'spongebob.fandom.com',
+  // 'tardis.fandom.com',
+  // 'thehungergames.fandom.com',
+  // '13reasonswhy.fandom.com',
+  // 'battlefield.fandom.com',
+  // 'dragonage.fandom.com',
+  // 'acecombat.fandom.com',
+  // 'borderlands.fandom.com',
+  // 'pixelgun-wiki.fandom.com',
+];
+
+
+export default Component.extend({
+  fetchService: service('fetch'),
+  logger: service(),
+  wikiVariables: service(),
+
+  isLoading: true,
+  isCrossWiki: false,
+  posts: null,
+  isNotCrossWiki: not('isCrossWiki'),
+
+  // fortunately we can compute the feeds path from articlePath (it has lang part)
+  feedsUrl: computed('wikiVariables.articlePath', function () {
+    return this.get('wikiVariables.articlePath').replace('/wiki/', '/f/');
```

Comment:
You don't need to use `get` implicitly. 
`this.wikiVariables.articlePath.replace('/wiki/', '/f/')`

Reply:
I've seen both approaches in the code. Are there any particular rules then to use `get` and when to omit it?

Reply:
You can use both. They work the same but without `.get()` is shorter :)

Reply:
Actually there is one difference
this.something.another will throw `Cannot read property 'another' of undefined` when `this.something` is undefined
this.get('something.another') will just return `undefined`

Reply:
Fixed in d3bb4c6